infrastructure/lambda/private-api.yaml

```diff
@@ -174,6 +174,45 @@ paths:
           Fn::Sub: arn:${AWS::Partition}:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/arn:${AWS::Partition}:lambda:${AWS::Region}:${AWS::AccountId}:function:${CommonStackName}-AuthorizationFunction/invocations
         passthroughBehavior: "when_no_match"
 
+  /addresses:
+    get:
+      parameters:
+        - in: header
+          name: "session_id"
+          schema:
+            type: string
+          required: true
+      responses:
+        "400":
+          description: "400 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "500":
+          description: "500 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "200":
+          description: "200 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PostcodeLookupResponse"
+      x-amazon-apigateway-request-validator: "Validate both"
+      x-amazon-apigateway-integration:
+        httpMethod: "POST"
```

Comment:
Is this supposed to be POST

Reply:
Yes, more details here - https://aws.amazon.com/premiumsupport/knowledge-center/api-gateway-lambda-template-invoke-error/